docs/source/customizations/coprocessor.mdx

```diff
@@ -580,48 +578,65 @@ A unique ID corresponding to the client request associated with this coprocessor
 <tr>
 <td>
 
-##### `serviceName`
+##### `body`
 
-`string`
+`string | object`
 
 </td>
 <td>
 
-The name of the subgraph that this coprocessor request pertains to.
+The body of the corresponding request or response.
 
-This value is present only for coprocessor requests from the router's `SubgraphService`.
+If your coprocessor [returns a _different_ value](#responding-to-coprocessor-requests) for `body`, the router replaces the existing body with that value. This is common when [terminating a client request](#terminating-a-client-request).
 
-**Do not return a _different_ value for this property.** If you do, the router treats the coprocessor request as if it failed.
+This field's type depends on the coprocessor request's [`stage`](#stage):
+
+- For `SubgraphService` stages, `body` is a JSON _object_.
+- For `RouterService` stages, `body` is a _string_.
```

Comment:
Because response chunks aren't pure serialized JSON I've just described this as a _string_ as opposed to a _JSON string_